content/projects/data-science-specific/data-wrangling/_index.md

```diff
@@ -65,29 +67,39 @@ df.apply(get_percentage, axis = 1) #axis=1 applies the function to all columns
 
 ### Instructions
 
-This data contains [personality scores](personality_scores.csv) for learners, plus the [department](departments.csv) they applied for.
+1. Your repo should contain everything needed to replicate your work. It's good practice to structure your files well, so we'll expect you to have a separate directory for "data" and "notebook", so that your final file structure looks something like this: 
 
-1. Import the dataset [personality_scores.csv](personality_scores.csv). Examine the data frame for duplicates (based on ID), and drop any duplicates that exist. Use an assert statement to check that the new data frame is the length of the unique entries of the original data frame.
+```
+├──data
+│   ├──departments.csv
+│   └──personality_scores.csv 
+├──notebook
+│  └──data_wrangling.ipynb
+├──README.md
+├──requirements.txt
+└──.gitignore 
+```
+2. Import the dataset [personality_scores.csv](personality_scores.csv). This data should be named `personality_df` Examine the data frame for duplicates (based on ID), and drop any duplicates that exist. Use an assert statement to check that the new data frame is the length of the unique entries of the original data frame. This new dataframe should be named `personality_score`
 
 Tip: An example assert statement is `assert 2*20=40` and it's a great way to check that your modification of the data was successful.
 
 2. Create new columns containing the total score of each of the personality test subscales. To do this, write a function (or functions) that will calculate the total score for each of the subscales (conscientiousness, emotional stability, openness to new experience, agreeableness, extraversion), as set out in [scoring](scoring.txt). In other words, for the conscientiousness total score, all items marked as belonging to 'conscientiousness' should be summed.
 
-The new data frame will look something like this:
+The new data frame will look something like this and it should be named `personality_score`:
```

Comment:
From the below example it seems like we'd like them to also clean up the column names - maybe let's be explicit about that?